### PR TITLE
Improve UI/UX for account aging column in 'Offers' section of GUI

### LINKED ISSUE #4735: Improve UI/UX for account aging column in 'Offers' section of GUI
### Background

This post was made as a result of me advocating for account signing on a couple of conversations which does not seem to be popular for reasons of privacy, introducing additional times to access harder limits,  and making it harder to establish a new payment methods.

Comments for reference: 

https://github.com/bisq-network/proposals/issues/264#issuecomment-719141219
https://github.com/bisq-network/proposals/issues/243#issuecomment-719319033

Therefore instead of advocating for account signing I am making a proposal to introduce some of the UI/UX benefits of payment methods that require account signing to payment methods that do not require account signing.

### Improve UI/UX for account aging in offers screen

Currently traders using payment methods that do not require signing are unable to see account age in the 'time since signing' column in the Offers Section.

Traders can see account ages by clicking the '🛈' next to each trade for more information. However traders cannot see this information when looking at the offers section as a whole.

The text and iconography shown in the 'time since signing' section could also benefit from some tweaks. Personally I find the repeated use of the '🛈' icon not helpful. I think the iconography would benefit from portraying the actual information it wants to communicated rather than showing a link to additional information. 

#### Why is not being able to see account age a problem?

Not being able to see account age at a glance makes has the following problems:

Traders are unable to quickly use age of account in their risk analysis when buying / selling
New users can be confused between risk analysis of payment methods that require signing vs those that do not
New users will not be able to clearly see that the account they are trading with is brand new.

When determining who to trade with on a Fiat/BTC transfer I think a significant number of users will take into account account age. Therefore it would improve the usability if this was shown.

#### Why is text and iconography in the offer section a problem?

Iconography can help aid users to make quick decisions by helping users associate symbols with specific information.  Iconography used to point out places where more information is available does not lead to quick decisions and makes it harder for users to differentiate between information.

#### Version

1.4.2

### Steps to reproduce

In Bisq client go to 'Market' > Sell BTC 
In Bisq client go to 'Market' > Buy BTC 

### Expected behaviour 

Traders using payment methods that do not require signing should be able to see the account age in the 'time since signing' column in the Offers Section.

Text and iconography in the offer section to help provide specific information to traders to quickly aid decision making

### Actual behaviour of the 'time since signing' column

FP 1

For the screenshot above I choose to put a '0' day account age at the top of the column. In the current design users would be unable to see that the top offer for buying BTC was from an account that was created on the same day.  This is currently a hinder to risk analysis.

### Proposed behaviour of the 'time since signing' column

FP 2

For the screenshot above I choose to put a '0' day account age at the top of the column. In this proposed design users would be able to see that the top offer for buying BTC was from an account that was created on the same day day.  This would improve risk analysis.

### Actual behaviour vs Proposed behaviour for the text and iconography in the offer section, including  mouse overs / pop ups.

Idea

**Current situation:**

Currently the text/ iconography tell traders when signed accounts have their limits lifted.
Currently the text/ iconography do not tell traders when accounts that do not require signing have their limits lifted.
Currently the '🛈' is used multiple times to mean more info. 
Currently N/A is used to represent instances where payment methods are not required to be signed 

**Proposed situation:**

- A clock icon '🕜︎' to represent waiting  for limits to be lifted
- For payment methods not required to be signed  'N/A' to be replaced with their account aged eg '30 days'.
- For payment methods not required to be signed  0-29 day old accounts to be given x 1 🗹. Text to show 'This account type does not require signing and was created less than 30 days ago (No limits lifted)'
- For payment methods not required to be signed  30-59 day old accounts to be given x 2 🗹 Text to show 'This account type does not require signing and was created less than 30 days ago (Some limits lifted)'
- For payment methods not required to be signed  60+ day old accounts to be given x 3 🗹 Text to show 'This account type does not require signing and was created less than 30 days ago (All limits lifted)'

_NB: The ticks above where just chosen as I was looking for something simple in the Windings font. I am sure there could be a better icons to use to achieve the objective._ 

### Negative impact of proposals?

- I am unsure how much time / resources they would take to implement (I tried to keep it simple)
- Privacy concerns. Making account ages more accessible does decrease privacy somewhat, but only by making more the account age more accessible. No additional information is shown than previous.
- My changes would make traders less likely to trade with users of 0-59 day payment methods accounts and more likely to trade with of 60 day plus payment methods.  

 
### Beneficial impact of proposals?

- Easier risk analysis for buyers and sellers
- Reduction in new users falling victim to scammers (Based on the assumption that more scams are committed with 0-59 day accounts, and on the assumption new users may be less likely to trade with accounts of that age if proposed changes are made)
- Better user experience for all traders 
- Increased confidence for new and existing users when making trades 
- Markets made up mostly of payment methods that do not require account signing eg GBP will see increased liquidity (this is my hypothesis and not based on any evidence)
- Payment methods that do not require account signing eg Faster Payment will see increased liquidity (this is my hypothesis and not based on any evidence). Payment methods that do not require signing numerous yet payment methods that require signing dominate the most popular used payment methods. I think a percentage of this can be attributed to usability and user experience, both of which would be improved with this poposal. For reference the current payment methods that [do not require signing](https://docs.bisq.network/payment-methods#account-signing) include the below.

### Payment methods affected by proposals?

- Advanced Cash
- Alipay
- Cash Deposit 
- Face-to-Face (F2F)
- Faster Payments
- HalCas
- Japan Zengin Furikomi      
- MoneyBeam (N26)
- MoneyGram      
- Perfect Money      
- PromptPay      
- Swish      
- US Postal Money Order      
- WeChat Pay      
- Western Union      
- TransferWise to TransferWise (when added)

### PR DESCRIPTION
**Pull request by ghost**
Showing 'x' days until limits lifted (signed account, pending)
Showing clock icon (signed account, pending)
Showing outlined check icon (aging accounts)
Showing account age (unsigned or aging accounts)
Changed column name from 'Signed since' to 'Account info'

Fixed bug when switching between altcoins and fiat order books the account info column was being added back to the table in the wrong position.

Fixes #4735 

### Screenshots

image

image

### CODE REVIEW

File: desktop/src/main/java/bisq/desktop/util/GUIUtil.java [hunk 1]
```diff
@@ -1115,6 +1115,8 @@ public static String getBsqInUsd(Price bsqPrice,
     }
 
     public static MaterialDesignIcon getIconForSignState(AccountAgeWitnessService.SignState state) {
+        if (state.equals(AccountAgeWitnessService.SignState.PEER_INITIAL))
+            return MaterialDesignIcon.CLOCK;
```
Comment (ghubstan) [inline desktop/src/main/java/bisq/desktop/util/GUIUtil.java:1119]: We (and the compiler) know, but there should(?) be a white space after the one line inner block of the if statement.

**Review by ghubstan — COMMENTED** [review]
utACK

File: desktop/src/main/java/bisq/desktop/util/GUIUtil.java (hunk 1 above)
Comment (ghost) [reply desktop/src/main/java/bisq/desktop/util/GUIUtil.java:1119]: thanks for the reminder, I've been warned before about my `if` statements!  :smiley:

**Review by ripcurlx — APPROVED** [review]
utACK

Thanks for taking care of this @jmacxx!

**Pull request merged**